### DISCUSSION TITLE
Straight to docs in docusarus 2?

### DISCUSSION THREAD

**Discussion by Quafadas** [facebook/docusaurus#7169] [Q&A] [answered]
Would this recipe still be expected to work with docusarus 2? 

https://v1.docusaurus.io/docs/en/site-creation#docs-landing-page

I'm struggling with it, can't get it to recognise the index.html in website/statc...

**Comment by slorber** [comment, marked as answer]
You can configure v2 home path `/` to be a doc:

https://docusaurus.io/docs/docs-introduction#docs-only-mode

**Comment by Quafadas** [comment]
Thankyou!
